### DISCUSSION TITLE
Merge CMILib and CMIPaperLib

### DISCUSSION THREAD

**Discussion by Sikatsu** [Zrips/CMI#5268] [Feature Requests]
Was wondering if these two could be merged, not sure why there has to be two different libraries.
Most plugins are able to to support both paper and spigot methods without the need of a library.

Thanks! ^^

**Comment by Zrips** [comment]
This library is specifically for paper branch methods, tho i get your idea and will try to figure out how to more properly implement paperlib into cmilib

**Reply by Sikatsu** [reply]
I understand, but it would be nice if these two can be merged into one library. Thanks so much!
